.github/workflows/build_and_test.yml

```diff
@@ -443,7 +443,7 @@ jobs:
     container:
       image: 369495373322.dkr.ecr.eu-central-1.amazonaws.com/build-tools:${{ needs.build-buildtools-image.outputs.build-tools-tag }}
       # for changed limits, see comments on `options:` earlier in this file
-      options: --init --shm-size=512mb --ulimit memlock=67108864:67108864 --cgroupns=private --privileged
+      options: --init --shm-size=512mb --ulimit memlock=67108864:67108864 --cgroupns=private --security-opt umask=/sys/fs/cgroup
```

Comment:
https://github.com/neondatabase/neon/actions/runs/7668433116/job/20901654848#step:2:79:

> `Error response from daemon: invalid --security-opt 2: "umask=/sys/fs/cgroup"`

Reply:
IT's a podman only feature (and I had a typo, it should be `umask`): https://neondb.slack.com/archives/C059ZC138NR/p1706280798542969?thread_ts=1706265241.134019&cid=C059ZC138NR

Reply:
> should be `umask`

`u_n_mask` -- yeah I was thinking of podman because I couldn't find this. Failed to notice that other thread, resolving this.